src/core/webview/webviewMessageHandler.ts

```
@@ -441,23 +392,12 @@ export const webviewMessageHandler = async (
 		case "exportCurrentTask":
-			await handleExportTask()
-			break
-		case "exportTaskToCloud": {
-			await provider.exportTaskToCloud()
-			break
-		}
-		case "importTaskFromCloudByUrl": {
-			const { url } = message.values || {}
-			if (url) {
-				await provider.importTaskFromCloudByUrl(url)
+			const currentTaskId = provider.getCurrentCline()?.taskId
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)